data/en/arrayfindallnocase.json

```diff
@@ -17,6 +17,13 @@
 	},
 	"links": [
 
+	],
+	"examples": [
+		{
+			"title": "Script Syntax",
+			"description": "Find the positions of a given string within a provided array regardless of case.",
+			"code": "var fruitArray = ['apple','banana','apple','orange','kiwi'];\nvar applePositions = arrayFindAllNoCase(fruitArray, 'APPLE');\nwriteDump(applePositions);",
```

Comment:
Just an idea: You could write the second "apple" different to the first one (like "aPpLe").